### PR TITLE
Dwr/arg support

### LINKED ISSUE #48: Support for --build-arg in build-image
It would be nice to be able to pass in aws environment variables generated by aws-vault to build-image without the credentials being stored in the image.

https://docs.docker.com/engine/reference/commandline/build/

### PR DESCRIPTION
**Pull request by rappdw**
fixes #48 

Add build_args and args support to dockerutils.cfg for `build-image` and `run-image`

### CODE REVIEW

File: scripts/build-image [hunk 1]
```diff
@@ -42,19 +42,21 @@ def build(image, image_name, image_tag, config, pull=False):
...
     if pull:
         pull_base = '--pull'
     if is_multistage(image):
         # if this is a multistage build and it follows the conventions, tag the builder image
         # otherwise, a prune will remove the layers used during the builder phase and subsequent
         # builds will take longer than required
         rc = image_operation(
-            'docker build {pull_base} --compress -t {image_name}-builder:{image_tag} -f docker/{image}/Dockerfile --target builder .'
-                .format(pull_base=pull_base, image_name=image_name, image=image, image_tag=image_tag))
+            'docker build {pull_base} --compress -t {image_name}-builder:{image_tag} -f docker/{image}/Dockerfile --target builder {args}.'
```
Comment (mikekwright) [inline scripts/build-image]: should there be a space between `{args}` and `.`?

File: scripts/run-image [hunk 2]
```diff
@@ -133,7 +129,8 @@ if __name__ == '__main__':
             'volumes': '',
             'ports': '',
             'cmd': args.command or '',
-            'init': init
+            'init': '--init',
```
Comment (mikekwright) [inline scripts/run-image:132]: Did we decide that we always want `--init`?

File: scripts/build-image (hunk 1 above)
Comment (rappdw) [reply scripts/build-image]: Yes

File: scripts/run-image (hunk 2 above)
Comment (rappdw) [reply scripts/run-image:132]: Conversation offline with Mike on this one...

**Pull request merged**